nat-lab/tests/telio.py

```diff
@@ -813,6 +790,13 @@ def get_features(self) -> Features:
         assert self._telio_features
         return self._telio_features
 
+    def get_adapter_type(self) -> TelioAdapterType:
+        if self._adapter_type is not None:
+            return self._adapter_type
+        if isinstance(self.get_router(), WindowsRouter):
+            return TelioAdapterType.WIREGUARD_GO_TUN
+        return TelioAdapterType.BORING_TUN
```

Comment:
I guess these two are written BoringTun and WireGuard, so why one of them is written with underscore and the other one without?

Reply:
If you check in the `.udl` it's `WireguardGoTun` and `BoringTun`, hence `WIREGUARD_GO_TUN` and `BORING_TUN`